pipeline_live/data/sources/alpaca.py

```diff
@@ -1,4 +1,5 @@
 import alpaca_trade_api as tradeapi
+from alpaca_trade_api import TimeFrame
```

Comment:
Should be able to import REST and then do `api = REST()`. This should keep the imports simpler.